### PR TITLE
Add var posterior kwargs, edit AR-HMM mean, and initial vLEM multiple subspace commit

### PR DESCRIPTION
**Pull request by davidzoltowski**

### CODE REVIEW

File: ssm/emissions.py [hunk 1]
```diff
@@ -541,7 +555,7 @@ def initialize(self, datas, inputs=None, masks=None, tags=None):
         yhats = [self.link(np.clip(d, .1, .9)) for d in datas]
         self._initialize_with_pca(yhats, inputs=inputs, masks=masks, tags=tags)
 
-    def hessian_log_emissions_prob(self, data, input, mask, tag, x):
+    def hessian_log_emissions_prob(self, data, input, mask, tag, x, Ez):
```
Comment (slinderman) [inline ssm/emissions.py]: Let's raise an Exception with an error message saying that we haven't supported multiple subspaces yet.

File: ssm/variational.py [hunk 2]
```diff
@@ -255,8 +255,13 @@ def _initialize_variational_params(self, data, input, mask, tag):
         # of the emissions invert function.
         J_diag = np.tile(1.0 / self.initial_variance * np.eye(D)[None, :, :], (T, 1, 1))
         J_lower_diag = np.zeros((T-1, D, D))
-        h = (1.0 / self.initial_variance) \
-            * self.model.emissions.invert(data, input=input, mask=mask, tag=tag)
+        if self.model.emissions.single_subspace:
+            h = (1.0 / self.initial_variance) \
+                * self.model.emissions.invert(data, input=input, mask=mask, tag=tag)
+        else:
+            # TODO smarter inversion with multiple subspace!
```
Comment (slinderman) [inline ssm/variational.py]: let's issue a warning for the multiple subspace initialization

File: ssm/observations.py [hunk 3]
```diff
@@ -694,7 +696,7 @@ def _compute_mus(self, data, input, mask, tag):
         mus = []
         for k, (A, b, V, mu0) in enumerate(zip(As, bs, Vs, mu0s)):
             # Initial condition
-            mus_k_init = mu0 * np.zeros((self.lags, D))
+            mus_k_init = mu0 * np.ones((self.lags, D))
```
Comment (slinderman) [inline ssm/observations.py]: Good catch!

File: ssm/emissions.py (hunk 1 above)
Comment (davidzoltowski) [reply ssm/emissions.py]: I replaced each assertion of single_subspace with an exception saying multiple subspaces are not supported yet.

File: ssm/variational.py (hunk 2 above)
Comment (davidzoltowski) [reply ssm/variational.py]: I added a warning that the posterior initialization has not been implemented for multiple subspaces.

**Pull request merged**